app/server/app/public/data/community/upper-content.json

```diff
@@ -0,0 +1,38 @@
+{
+  "aquaticLife": {
+    "body": "<p>EPA, states, and tribes monitor and assess water quality to determine the impact of impairments on plants and animals living in the water.</p><p>Plants and animals depend on clean water. Impairments can affect the quality of water, which can have adverse effects on plants and animals living in the water.<span data-show-less-more> The condition of a waterbody is dynamic and can change at any time, and the information in How’s My Waterway should only be used for general reference. If available, refer to local or state real-time water quality reports.</span></p>",
+    "disclaimer": "<p>Users of this application should not rely on information relating to environmental laws and regulations posted on this application. Application users are solely responsible for ensuring that they are in compliance with all relevant environmental laws and regulations. In addition, EPA cannot attest to the accuracy of data provided by organizations outside of the federal government.</p>"
+  },
+  "drinkingWater": {
+    "body": "<p>Drinking water comes from aquifers, streams, rivers, and lakes. Approximately 150,000 <span data-glossary-term data-term=\"Drinking Water System Type\">public water systems</span> provide drinking water to most Americans. Under the Safe Drinking Water Act, EPA sets standards for drinking water quality and with its partners implements various technical and financial programs to ensure drinking water safety.<span data-show-less-more> Drinking water quality is based on water that has been treated to ensure it is ready for human consumption which differs from raw water in streams and reservoirs.</span></p>"
+  },
+  "eatingFish": {
+    "body": "<p>EPA, states, and tribes monitor and assess water quality to determine if fish and shellfish are safe to eat.</p> <p>Eating fish and shellfish caught in impaired waters can pose health risks. For this watershed, be sure to look for posted fish advisories or consult your local or state environmental health department for <span id=\"eating-fish-state-links\" /><span data-show-less-more> The information in How’s My Waterway about the safety of eating fish caught recreationally should only be considered as general reference. Please consult with your state for local or state-wide fish advisories.</span></p>",
```

Comment:
```suggestion
    "body": "<p>EPA, states, and tribes monitor and assess water quality to determine if fish and shellfish are safe to eat.</p> <p>Eating fish and shellfish caught in impaired waters can pose health risks. For this watershed, be sure to look for posted fish advisories or consult your local or state environmental health department for <span id=\"eating-fish-state-links\"></span><span data-show-less-more> The information in How’s My Waterway about the safety of eating fish caught recreationally should only be considered as general reference. Please consult with your state for local or state-wide fish advisories.</span></p>",
```

I noticed an issue where the last couple of sentences, for eating fish, were not showing up. Explicitly including a closing span tag seems to fix it.

Reply:
Good catch!